iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/TieLineAdderImpl.java

```diff
@@ -124,6 +124,11 @@ public TieLineAdderImpl add() {
             return TieLineAdderImpl.this;
         }
 
+        public TieLineImpl.HalfLineImpl build() {
```

Comment:
Can this be package-private?

Reply:
I change the visibility to private: this method should be called only by the enclosing TieLineAdder instance